### ISSUE TITLE
Finish progress tracking mechanism

### ISSUE THREAD

**Issue by Sean-Meyers** [Sean-Meyers/The-Fix#7]
The prototype requires us to "Include a progress tracker against the elements you planned in your Game Concept Document, what is complete, what is yet to be implemented, what have you decided to scrap (or add). Updated timeline/schedule optional but encouraged."

- We should add more unofficial sub-milestones to our milestones. Maybe we can use CMPUT250's setup as inspiration?
 - We should have a feature list and feature cut list in our project board, assigned to milestones, possibly with priority indications attached (?).
 - We'll need to invite Kenzie to this repo so she can see it and mention it somewhere in our submission somehow, so she knows.

This will also allow us to more formally state what our planned feature and stuff are. Things to include (we can cut this if we're on the crunch)
- Story beats and stuff from the design doc.
- Mechanics
- Level design plans (rough/loose goals are fine)
- Art/audio/writing asset plans/goals/list thing
   - BGM list
   - Sfx list
   - (maybe) Story elements like characters and stuff idk
   - Art asset list

Basically just, what our game should have implemented by when.
